static/js/components/FollowupRequestLists.jsx

```diff
@@ -84,7 +84,12 @@ const FollowupRequestLists = ({
   followupRequests,
   instrumentList,
   instrumentFormParams,
+  totalMatches,
+  handleTableChange = false,
+  pageNumber = 1,
+  numPerPage = 10,
   showObject = false,
+  serverSide = false,
```

Comment:
What does `serverSide` do?

Reply:
This is checking whether pagination occurs in MUI or in the handler.

Reply:
ok, got it.